### ISSUE TITLE
Replace light pictures with dark pictures when dark mode is on.

### ISSUE THREAD

**Issue by vldmrkl** [thexande/SwiftAlgorithms#5]
Currently, if you switch to dark mode, the illustrations still have light backgrounds, which doesn't really fit the dark theme. 

I'd be happy to help with it.

**Comment by thexande** [comment]
@klymenkoo Indeed, this is a tough problem considering all of the assets are present in the RWC algorithm club repo. One solution would be to invert the image colors in dark mode. What was your idea?
